### ISSUE TITLE
Make sure that for each montage `i` on o2, there exists a raw montage folder at `/mnt/emlode??_tank?/$i`. Some may have been moved to subfolders. If so, fish them out.

### ISSUE THREAD

**Issue by jasper-tms** [jasper-tms/righty#5]

**Comment by jasper-tms** [comment]
To deal with the lostandfound sections, whose folders got renamed on o2 but not on the emlodes, I'll need to look at the name of the `*_finished.json` inside each o2 folder and then look on the emlodes for a folder with a name that matches the json, because that json's filename matches the original acquisition folder name. Something like:

    for i in 2*; do (cd $i && ls -ld /mnt/emlode*/$(ls *json | sed 's/_finished.json//')); done

**Comment by jasper-tms** [comment]
After finding some lostandfound sections that were in subfolders, I think I have everything. Running this to check:
```
rm emlode_locations.lst notfound.lst
for i in $(cat ../intersection/alignmentV1/everything.lst); do
    echo $i $(ls -d /mnt/emlode*/$(cd $i && ls *json | sed 's/_finished.json//')) 1>> emlode_locations.lst 2>> notfound.lst
done
```

**Comment by jasper-tms** [comment]
Yep, all set. See `/n/groups/htem/temcagt/datasets/righty_r1062/imaging/emlode_locations.lst` for a full list of which emlode each montage came from.

**Comment by jasper-tms** [comment]
Hmm had two sections, `210829183442_righty_r1062_6615` and `210829215656_righty_r1062_6747`, where the `*_finished.json` filename wasn't right and so they weren't found, but I messed up where I put the `2>> notfound.lst` in the for loop above so those error messages went to the terminal instead of the text file. I guess that redirect needs to go inside the `ls` parentheses. Anyway, I manually fixed the json filenames and then added their emlode location to `emlode_locations.lst`. All 12058 rows now have an emlode entry
